tmail-backend/blob/blobid-list/src/main/java/com/linagora/tmail/blob/blobid/list/cassandra/BlobIdListTable.scala

```diff
@@ -0,0 +1,6 @@
+package com.linagora.tmail.blob.blobid.list.cassandra
+
+object BlobIdListTable {
+  val TABLE_NAME: String = "blobId_list"
```

Comment:
I prefer avoid weird characters like `_` in cassandra tables -> I am paranoid.

I prefer `blobIdList` and `blobId`.

Reply:
Before make Cassandra name, I have consulted the current tables.
I saw many table name has `_` characters 
![image](https://user-images.githubusercontent.com/81145350/118215246-61e10700-b49b-11eb-9dbb-f79a96548f42.png)

Reply:
Ok, forget my comment, then it is fine.